core/templates/dev/head/components/forms/ApplyValidationDirective.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
2014 -> 2019

Ditto elsewhere where new files are added due to refactor.

Reply:
Done